### ISSUE TITLE
Relation for light-verb constructions

### ISSUE THREAD

**Issue by ftyers** [ftyers/docs#10]
AM has `compound:lvc` for dealing with light verb constructions such as "темекі тарт-" and "гитара тарт-". 

I'm not sure about this. We would probably analyse these as simple verb + object combinations. 

We would argue for lexicalising light-verb constructions, e.g. "пайда бол-", although I'm not sure about the others. Note that: In order to have a reliable analysis we would need to have these listed either way.

A couple of things to take into account:
1. In some cases the noun part of the construction could be considered to be in indefinite accusative. So, one test for if it is a light verb construction would be if you can have another object (either in indefinite or definite accusative, or in another case, e.g. dative).
2. Can other words intervene between the noun and the light verb ?

We would certainly not like to simply determine light-verbness based on translation. What do people think ?

**Comment by jonorthwash** [comment]
Is there some way you could argue that "пайда бол" is light verb construction and "темекі тарт-" is not?

Also, verb constructions like these, if analysed analytically, could sometimes be considered adjective + verb instead of noun + verb.  E.g., "кабыл ал-" takes a subject and an accusative direct object, so maybe кабыл is actually an adjective or an adverb instead of a noun?  I guess it's hard to tell.

**Comment by jonorthwash** [comment]
There's another type of compound verb construction in Kazakh where it's easy to tell the role of the noun.  Examples include "бас айнал-" ("be dizzy"), "көңіл қал-" ("become disillusioned"), "мойынға ал-" ("accept responsibility [for..]").  For all of these verbs, the subject/object normally takes possessive morphology, and they're fairly analytical, despite the meanings not being entirely analytical.

These first two examples also have forms where the verb is causative and the subject becomes accusative.

**Comment by jonorthwash** [comment]
See other examples (in Kyrgyz) here: http://wiki.firespeaker.org/Kyrgyz_phrasal_verbs#N_.2B_V

**Comment by makazhan** [comment]
Just like "қабыл ал-", "жоқ/бар қыл-" (destroy/create) take a subject and accusative object and "жоқ" and "бар" seem to be adjectives there.
Also, other words can intervene between a nominal and a verb: "темекіні асықпай тартты" (smoked non-hastily)

**Comment by jonorthwash** [comment]
Agreed.  These are good arguments for treating "phrasal verbs" like темекі тарт- as two words with the appropriate dependency relation (in this case, `nobj(тарт, темекі)`).

As far as "phrasal verbs" like қабыл ал- and жоқ/бар қыл-, if we were to do the same, how would it work?  Relations like `amod(ал, қабыл)` or `advmod(ал, қабыл)` don't seem right.  I wonder if there any of these that take a subject and object where the head verb doesn't also take a subject or an object.  I.e., can you have a "phrasal verb" of different valency created using this strategy?
